packages/angular/projects/cds-angular/package.json

```
@@ -8,8 +8,8 @@
     "url": "https://github.com/vmware/clarity.git"
   },
   "peerDependencies": {
-    "@angular/common": "^9.0.0",
-    "@angular/core": "^9.0.0",
+    "@angular/common": "^11.0.0",
+    "@angular/core": "^11.0.0",
```

Same as the line above:
```suggestion
    "@angular/core": "9.x || 10.x || 11.x",
```